### ISSUE TITLE
The yt-watchme project is not for live stream?

### ISSUE THREAD

**Issue by ghost** [youtube/yt-watchme#33]
The yt-watchme project I tried to make it work on my ANDROID device has a lag(or latency) of almost ~1min.

If I start live stream from my device, the video at youtube events site will remain idle(shows black screen with the title.) for about a minute and then it will start streaming the 1 minute old data to the youtube site.

And the process continues, 

1.) the video will always be 1 minute late.
2.) the speed of the video is very fast by default.
3.) the audio will not work, even after compiling the lame according to the provided Android.mk file.
4.) is there anyway to @iulukaya to prove that this his project is worth the title of live streaming?

I am not asking for the code, but how to make the live stream possible with this small application.

**Comment by garyjamessilva** [comment]
Hi,

This project / app does provide 'live streaming' with Youtube, once you get
all the dependencies compiled, but not necessarily with audio.

The ~1 minute delay is due to YouTube buffering the incoming video before
broadcasting it, the long delay is not due to this app.

Try comparing to (Sony) 'Live on Youtube' app if possible, this is
currently giving me 3 to 5 minute delay on a slow broadband connection.

Regards

**Comment by achinverma** [comment]
Hi @sai-cool , you mentioned that **the speed of the video is very fast by default.**  , i am facing the same issue when i try to play the recorded Live stream in my yt-channel , please anyone guide me how to set the video speed to normal ?

**Comment by hwrrohit** [comment]
Hi ,  the speed of the video is very fast by default. , i am facing the same issue when i try to play the recorded Live stream in yt-wathcme example , please anyone suggest me how to set the video speed to normal ?

thanks

**Comment by nileshgangadhar** [comment]
In yt-watchme getting some problems like 
com.google.api.client.googleapis.json.GoogleJsonResponseException: 403 Forbidden
                                                                    {
                                                                      "code": 403,
                                                                      "errors": [
                                                                        {
                                                                          "domain": "youtube.liveBroadcast",
                                                                          "message": "Stream is inactive",
                                                                          "reason": "errorStreamInactive",
                                                                          "extendedHelp": "https://developers.google.com/youtube/v3/live/docs/liveBroadcasts/transition"
                                                                        }
                                                                      ],
                                                                      "message": "Stream is inactive"
                                                                    }

I have enable the streaming status but its not working plz Suggest me how to solve this issue.

**Comment by kokphinsheng** [comment]
T.k

Sent from Yahoo Mail on Android 

  On Wed, Oct 12, 2016 at 22:08, nileshgangadharnotifications@github.com wrote:  
In yt-watchme getting some problems like 
com.google.api.client.googleapis.json.GoogleJsonResponseException: 403 Forbidden
 {
 "code": 403,
 "errors": [
 {
 "domain": "youtube.liveBroadcast",
 "message": "Stream is inactive",
 "reason": "errorStreamInactive",
 "extendedHelp": "https://developers.google.com/youtube/v3/live/docs/liveBroadcasts/transition"
 }
 ],
 "message": "Stream is inactive"
 }

I have enable the streaming status but its not working plz Suggest me how to solve this issue.

**Comment by Kirancloudnix** [comment]
@sai-cool Can you help on how to make it work?
Request you to reach out to me on kiran@cloudnix.com

**Comment by MuthuEphron** [comment]
Hi guys, 
Pleas help this issues #53

**Comment by Muthukumar39** [comment]
Hi guys,
Pleas help this issues #53

**Comment by nileshgangadhar** [comment]
hey hi ,
  your network is slow that's why your getting this issue.

**Comment by MuthuEphron** [comment]
@nileshgangadhar  First Thanks for your reply. Please read the questions carefully.I recorded the live streaming in portrait mode in mobile but the youtube displayed in opposite direction, can u please take a demo record the live stream in portrait mode

**Comment by nileshgangadhar** [comment]
sorry i didn't tried in portrait mode



Thanks & regards,
Nilesh Gangadhar.
9404697050

**Comment by EdsonPAndrade** [comment]
I can't get mine to build...Help please

Sent from my iPhone

On Apr 15, 2017, at 2:35 PM, nileshgangadhar <notifications@github.com<mailto:notifications@github.com>> wrote:

sorry i didn't tried in portrait mode



Thanks & regards,
Nilesh Gangadhar.
9404697050

**Comment by nileshgangadhar** [comment]
yeah sure i will send you complete code please send me your mail id




Thanks & regards,
Nilesh Gangadhar.
9404697050

**Comment by EdsonPAndrade** [comment]
Katem2330@gmail.com<mailto:Katem2330@gmail.com>

Sent from my iPhone

On Apr 16, 2017, at 12:46 AM, nileshgangadhar <notifications@github.com<mailto:notifications@github.com>> wrote:

yeah sure i will send you complete code please send me your mail id




Thanks & regards,
Nilesh Gangadhar.
9404697050

**Comment by EdsonPAndrade** [comment]
Just checking if you was able to send me it. So I can start working on some of the issues. Katem2330@gmail.com<mailto:Katem2330@gmail.com>

Sent from my iPhone

On Apr 16, 2017, at 12:46 AM, nileshgangadhar <notifications@github.com<mailto:notifications@github.com>> wrote:

yeah sure i will send you complete code please send me your mail id




Thanks & regards,
Nilesh Gangadhar.
9404697050
